### DISCUSSION TITLE
Cant set Min X/Y on Ender 3 S1

### DISCUSSION THREAD

**Discussion by AlciAJ** [mriscoc/Ender3V2S1#1290] [Q&A]
I’ve got an Ender 3 S1 with a 235 by 235 bed size. Just updated to the latest pro firmware, but ran into an issue during setup. No matter what values I punch in for my min X/Y, my printer head insists on wandering off the bed. I’ve even tried some wild adjustments, but nothing. Feeling a bit stuck here. Any ideas?

**Comment by mriscoc** [comment]
Just follow: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

**Reply by AlciAJ** [reply]
Well that's the thing, I was following along and when I was setting my print head to my new "(0,0)," nothing changed. The print head still went off the bed. Even with extremely positive and negative values.
